libc/utils/docgen/api.yml

```diff
@@ -0,0 +1,107 @@
+ctype.h:
+  functions:
+    isalnum:
+      returns: int
+      params:
+        - int
```

Comment:
Having the function signature encoded here makes more sense for replacing libc-hdrgen, but isn't strictly required for documentation purposes (at the moment; perhaps we could generate something akin to very very minimal `man` pages, eventually).

Reply:
done in https://github.com/llvm/llvm-project/pull/87682/commits/29213d3706085000192a4b2259c346e50c5a0ffc